charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         - {{ .Values.apiserver.audit.logPath }}
         {{- end}}
         - --admission-control
```

Comment:
is this already kube 1.10 based? `--admission-control` will be deprectated in favor of `--enable-admission-plugins` to selectively enable plugins. The `AdmissionOptions` in 1.10 has a `RecommendedPluginOrder` selection of plugins, pre-set with the recommended list for aggregated apiservers.

Reply:
@sttts No, Service Catalog is currently based on 1.9, but thanks for the useful information, I will definitely try to migrate it on 1.10 soon.

Reply:
@sttts thanks for the pointer. good to have defaults. Is the underlying 1.10 lib of apistuff tagged and released?

Reply:
Yes, tagged with the usual automatic tags: https://github.com/kubernetes/apiserver/releases/tag/kubernetes-1.10.0-beta.2